Tools/GenerateWebsiteData.ps1

```diff
@@ -195,6 +288,8 @@ $productInfoLookup = @{
     "Gce"   = @{ name = "Google Compute Engine"; shortName = "google-compute-engine"; resources = @() }
     "GcSql" = @{ name = "Google Cloud SQL";      shortName = "google-cloud-sql"     ; resources = @() }
     "Gcd"   = @{ name = "Google Cloud DNS";      shortName = "google-cloud-dns"     ; resources = @() }
+    "Gcps"   = @{ name = "Google Cloud PubSub";      shortName = "google-cloud-pubsub"     ; resources = @() }
```

Comment:
In a perfect world these two changes would be in a separate CL so that the change description would match the code being updated. However, don't worry about it.